curryreader.py

```diff
@@ -179,13 +179,13 @@ def read(inputfilename='', plotdata = 1, verbosity = 2):
     nASCII      = int(a[5]  + a[int(5 + nt / 2)])
     nMultiplex  = int(a[6]  + a[int(6 + nt / 2)])
     fSampleTime =     a[7]  + a[int(7 + nt / 2)]
-
-    datainfo = { "samples" : nSamples, "channels" : nChannels, "trials" : nTrials, "samplingfreq" : fFrequency }
-    log.info('Number of samples = %s, number of channels = %s, number of trials/epochs = %s, sampling frequency = %s Hz', str(nSamples), str( nChannels), str(nTrials), str(fFrequency))
                     
-    if fFrequency == 0 or fSampleTime != 0:
+    if fFrequency == 0. or fSampleTime != 0.:  # BUG (order) FIXED

```

Comment:
Just so I'm clear, the bug here was adjusting `fFrequency` *after* it was entered into the `datainfo` dict, right? If so, let's avoid changes to this line, as there wasn't anything broken about it before, and we should try to minimize the diff with upstream so it's easier to pull in any upstream changes down the road.

```suggestion
    if fFrequency == 0 or fSampleTime != 0:
```